src/seeds.history.cpp

```diff
@@ -113,202 +126,236 @@ void history::trxentry(name from, name to, asset quantity) {
     return;
   }
 
-  bool from_is_organization = from_user->type == "organisation"_n;
+  uint64_t day = utils::get_beginning_of_day_in_seconds();
+  daily_transactions_tables transactions(get_self(), day);
+
+  uint64_t transaction_id = transactions.available_primary_key();
+  uint64_t timestamp = eosio::current_time_point().sec_since_epoch();
+
+  bool from_is_organization = from_user -> type == "organisation"_n;
+  bool to_is_organization = to_user -> type == "organisation"_n;
+
+  int64_t transactions_cap = int64_t(config_get("qev.trx.cap"_n));
+  int64_t max_transaction_points_individuals = int64_t(config_get("i.trx.max"_n));
+  int64_t max_transaction_points_organizations = int64_t(config_get("org.trx.max"_n));
+
+  double from_trx_multiplier = (
+    from_is_organization ? 
+    std::min(max_transaction_points_organizations, quantity.amount) : 
+    std::min(max_transaction_points_individuals, quantity.amount)
+  ) / 10000.0;
   
-  if (from_is_organization) {
-    org_tx_tables orgtx(get_self(), from.value);
-    orgtx.emplace(_self, [&](auto& item) {
-      item.id = orgtx.available_primary_key();
-      item.other = to;
-      item.in = false;
-      item.quantity = quantity;
-      item.timestamp = eosio::current_time_point().sec_since_epoch();
+  double to_trx_multiplier = std::min(max_transaction_points_organizations, quantity.amount) / 10000.0;
```

Comment:
Why is this divided by 10000?

Should this not be called "capped_amount" - seems like that's what it is the quantity, capped to the limit ?

Reply:
I divided it by 10000 because I mimicked the way transaction points were originally calculated in harvest. Yes, capped_amount is a better name, I will change it

Reply:
Right so points are assigned on Seeds traded by the amounts are x10,000 ... makes sense.